benchmarks/GeneratedHydratorBenchmark/AbstractHydrationBench.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace GeneratedHydratorBenchmark;
+
+use GeneratedHydrator\Configuration;
+
+abstract class AbstractHydrationBench
+{
+    protected $hydrator;
```

Comment:
God I hate to state the obvious, but I'll do it in order to be respectful to standards.